dvc/utils/cli_parse.py

```diff
@@ -0,0 +1,45 @@
+from collections import defaultdict
+from typing import Any, Dict, Iterable, List
+
+
+def parse_params_from_cli(path_params: Iterable[str]) -> Dict[str, List[str]]:
```

Comment:
Maybe `parse_params` is enough? Information about `cli` is in import.

Reply:
same for `load`

Reply:
Will do in the next PR. Thanks @pared.